src/models/playspot.js

```diff
@@ -0,0 +1,26 @@
+export const UPDATE_PLAYSPOT = 'UPDATE_PLAYSPOT'
+
+export default {
+  name: 'playspot',
+  state: {
+    // TODO: Figure out how to set this initially
+    current: null
+  },
+  mutations: {
+    UPDATE_PLAYSPOT(state, payload) {
+      state.current = payload
+    },
+  },
+  actions: {
+    playNext({commit}) {
+      // TODO: Get the next playspot
+      const next = null;
```

Comment:
to get the next/previous playspot, I think we can use rootState.toys.paths, it's like:

```js
{Button: [{path: '/Button/with_text', type: 'with text'}]}
```

You can add a getter to flatten it.